duckduckgo/macos-browser
# Update autoconsent to 3.0.2 to fix issue with Sourcepoint CMP

Task/Issue URL: https://app.asana.com/0/1201844467387842/1203144940444659/f
Tech Design URL: n/a

**Description**:

This is a fix to the issue mentioned in the Asana task, i.e. a wrongful detection of a CMP on derstandard.at which causes the UI dialogue for the first-time autoconsent setup to show up but will then not produce any visible effect as the CMP can't be handled.

**Steps to test this PR**:
1. Go to derstandard.at with a **fresh** (!) install of the MacOS browser.
1. Ensure that the dialogue asking the user whether to enable autoconsent does **not** show up.

**Testing checklist**:
